### PR TITLE
fix invalid dragged attributes

### PR DESCRIPTION
**Pull request by sarracini**

### CODE REVIEW

**Comment by sarracini** [comment]
@marcoscaceres r?

File: css/newTab.css [hunk 1]
```diff
@@ -153,7 +153,7 @@ input[type=button] {
   pointer-events: none;
 }
 
-.newtab-site[dragged] {
+.newtab-site[data-dragged] {
```
Comment (marcoscaceres) [inline css/newTab.css]: so, this should match on:

``` css
.newtab-site[data-dragged=true] {
```

File: css/newTab.css [hunk 2]
```diff
@@ -300,7 +300,7 @@ input[type=button] {
   opacity: 1;
 }
 
-.newtab-control[dragged] {
+.newtab-control[data-dragged] {
```
Comment (marcoscaceres) [inline css/newTab.css]: as above

File: js/drag.js [hunk 3]
```diff
@@ -109,8 +109,8 @@
      * @param {Site} aSite The site that's being dragged.
      */
     end(aSite) {
-      for (let node of gGrid.node.querySelectorAll("[dragged]")) {
-        node.removeAttribute("dragged");
+      for (let node of gGrid.node.querySelectorAll("[data-dragged]")) {
+        delete node.data.dragged;
```
Comment (marcoscaceres) [inline js/drag.js]: Then just set it to false.

File: css/newTab.inc.css [hunk 4]
```diff
@@ -127,13 +127,13 @@
 
 .newtab-cell:not([ignorehover]) .newtab-control:hover ~ .newtab-link,
 .newtab-cell:not([ignorehover]) .newtab-link:hover,
-.newtab-site[dragged] {
+.newtab-site[data-dragged] {
```
Comment (marcoscaceres) [inline css/newTab.inc.css]: as above... and the one below too...

**Comment by marcoscaceres** [comment]
R+

**Pull request merged**